src/main/java/pl/chojnacki/lesson5/homework/task3/Addition.java

```diff
@@ -0,0 +1,7 @@
+package pl.chojnacki.lesson5.homework.task3;
+
+public class Addition {
+    public static double add (double number1, double number2) {
```

Comment:
założmy, że teraz chce użyć tych metod w jednej klasie - Calculator np w ten sposób:
```

Calculator {

calculate(Operation operation, number1, number2){
mapWithOperatios.get(Operatoin).calculate(number1,number2);
}

}

```

więc w mapie/liście/tablicy chcę trzymać listę tych operacji. Jak to zrobić, aby te klasy, które tutaj napisałeś można było sprowadzić do jednego zbioru? Za pomocą czego najlepiej tutaj wyznaczyć coś co mają wspólnego? ;)